src/main/kotlin/com/exactpro/th2/conn/dirty/tcp/core/api/impl/HandlerContext.kt

```diff
@@ -32,6 +33,7 @@ class HandlerContext(
     private val channelFactory: ChannelFactory,
     private val getDictionary: (DictionaryType) -> InputStream,
     private val sendEvent: (Event) -> Unit,
+    private val grpcRouter: GrpcRouter
```

Comment:
Let's pass a method reference instead of the whole router

Reply:
How? We don't have common interface for service classes

Reply:
> How?

We can do smth like this, but we'll have to do an unchecked cast later

```suggestion
    private val getService: (Class<out Any>) -> Any
```